packages/spruce-utils/lib/graphql.js

```diff
@@ -197,14 +197,14 @@ export class GraphQLClient {
 			if (e.networkError || e.graphQLErrors) {
 				const graphQLErrors = get(e, 'graphQLErrors', [])
 				const networkErrors = get(e, 'networkError.result.errors', [])
-
 				const errors = networkErrors.concat(graphQLErrors)
 
 				throw new SpruceWebError(
 					`GraphQL Request Failed (${operationType} ${definitionNames.join(
 						'/'
 					)}). Reasons: [${errors.map(error => error.reason).join(', ')}]`,
 					{
+						friendlyReason: errors && errors[0] && errors[0].friendlyReason,
```

Comment:
You should pass all `friendlyReasons` and defer to the user of the error to decide what to do with it. Engineers should have all of them or none of them.